webhook.rb

```diff
@@ -0,0 +1,42 @@
+require 'sinatra'
```

Comment:
your file must be in lib/microkube/server.rb
You can commit a Rackup file to have any web server running it
also include puma or thin in Gemfile to install a webserver